MutagenMerger.CLI/MutagenMerger.CLI.csproj

```diff
@@ -2,13 +2,13 @@
 
     <PropertyGroup>
         <OutputType>Exe</OutputType>
-        <TargetFramework>net6.0</TargetFramework>
+        <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
```

Comment:
I think for end executables it's better to keep it on single TargetFramework.   The multiple targeting is mostly only useful for libraries, which want to offer some flexibility for downstream users